core/alsp_src/library/html_tokens.pro

```diff
@@ -576,8 +576,8 @@ next_token(0'&, S, T, NxtC)
 	((NNxtC=0' ; NNxtC=0'	; NNxtC=10; NNxtC=13) ->
 		Cs = [], NxtC = NNxtC
 		;
-		dispatch_read_to_semi(NNxtC, S, Cs, NxtC),
-		get_code(S,NxtC)
+		dispatch_read_to_semi(NNxtC, S, Cs, NxtC)
+%		,get_code(S,NxtC)
```

Comment:
Better to just delete the bad line -- if it were later uncomment, the bug will return.